### ISSUE TITLE
[Feature]: Adding a Neural Machine Translation (NMT)-Based Autocorrect Tutorial in project page website.

### ISSUE THREAD

**Issue by IkkiOcean** [ML-Fusion-Lab/ML-Fusion-Lab-Website#1002]
### Is your feature request related to a problem? Please describe.

I would like to propose the creation of a tutorial that focuses on building an autocorrect system using Neural Machine Translation (NMT) techniques. The tutorial will guide users through the process of developing a basic autocorrect functionality, leveraging the power of NMT to improve text input accuracy in applications.

### Describe the solution you'd like.

- Introduce the concepts of Neural Machine Translation and its application in autocorrect systems.
- Provide step-by-step instructions for setting up the environment and necessary libraries.
- Include code snippets and explanations for training an NMT model for autocorrect purposes.

### Checklist

- [X] I have checked the [Existing Issues](https://github.com/ML-Fusion-Lab/ML-Fusion-Lab-Website/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/ML-Fusion-Lab/ML-Fusion-Lab-Website/blob/main/contributing.md)
- [X] I want to work on this issue. (optional)

**Comment by vivekvardhan2810** [comment]
@IkkiOcean assigning this issue to you.

Make sure to complete this.

**Comment by vivekvardhan2810** [comment]
i will be active till tomorrow evening after that this project won't be active and me also.

**Comment by IkkiOcean** [comment]
@vivekvardhan2810 oh okay. well I working on the tutorial since morning I hope it comes out good.

**Comment by vivekvardhan2810** [comment]
@IkkiOcean hoping it to come out good!.

**Comment by IkkiOcean** [comment]
@vivekvardhan2810 hey before you go offline and this repo too. Can I add how to create neural networks from scratch like ReLu function, sigmoid function and their gradient descents? it would be good tutorial for someone starting their journey in Neural Networks since it will give insight on the functions which we use very casually from libraries like tensorflow... Do tell me your thought

**Comment by vivekvardhan2810** [comment]
@IkkiOcean means that you will create another tutorial like how to create neural networks from scratch like ReLu function, sigmoid function and their gradient descents in the projects sections.

is it.
